lib/events/managerCardListingEvents.js

```diff
@@ -0,0 +1,7 @@
+if (Meteor.isClient) {
+	Template.ManagerCardListing.events({
+		'click .modReceivedOfferTrigger': function () {
+			Session.set("listingSelected", this)
```

Comment:
Mixed double and single quotes.<br>Missing semicolon.<br>'Session' is not defined.